apps/wallet/src/shared/plausible.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import Plausible from 'plausible-tracker';
+
+// NOTE: The url of Sui wallet Chrome extension:
+// https://chrome.google.com/webstore/detail/sui-wallet/opcgpfmipidbgpenhmajoajpbobppdil
+const WALLET_URL = 'chrome-extension://opcgpfmipidbgpenhmajoajpbobppdil';
+
+const PLAUSIBLE_ENABLED = process.env.NODE_ENV !== 'development';
```

Comment:
We could check to be `production` I think it's always set when we build for prod. And it will not run for tests as well